third_party/proton/dialect/include/Dialect/Proton/IR/ProtonDialect.td

```diff
@@ -0,0 +1,20 @@
+#ifndef PROTON_DIALECT
+#define PROTON_DIALECT
+
+include "mlir/IR/OpBase.td"
+
+def Proton_Dialect : Dialect {
+  let name = "proton";
+  let cppNamespace = "::mlir::triton::proton";
+
+  let description = [{
+    Proton Dialect provides core ops for building third-party compiler-based
+    performance profiling and analysis tools.
+  }];
+
+  let dependentDialects = [
+    "mlir::LLVM::LLVMDialect"
```

Comment:
Do you have to make it depend on the `LLVMDialect`? Or we can just make the lowering pass depend on the `LLVMDialect`

Reply:
Good question. Are we going to allow Proton dialect to interact with LLVM dialect, or strictly on top of that, making LLVM dialect a lowering/conversion target? If Proton dialect would talk to LLVM dialect, we need some Proton operators that take LLVM data types at some point, thus our dialect would need to depend on it. Otherwise, we could simply rely on passes to depend on it.

Reply:
> strictly on top of that

I was thinking about this

Reply:
We could start from "strictly on top of that" and later revisit this question when needed. So I will remove it for now and we can always add it back.